Tasks/ANT/README.md

```diff
@@ -40,3 +28,4 @@ Use the next options to manage your `ANT_HOME` and `JAVA_HOME` attributes
 
 - **JDK Architecture :** Select the approriate JDK Architecture. By default it is set to `x86`
 
+**With TFS Dev15 and Sprint 107 deployment of VSTS, we are discontinuing 'Code Coverage' functionality. Please enable Code Coverage in your Ant build.xml file manually**
```

Comment:
>TFS Dev15 and Sprint 107 [](start = 7, length = 24)

the numbers needs to be consistent with other places where we mention these. do we call it tfs dev15?